plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobCard.qml

```diff
@@ -0,0 +1,179 @@
+ * with '// FIXED-LINE-HEIGHT:'.
+ */
+Item
+{
+    id: base
+
+    // The print job which all other data is derived from
+    property var printJob: null
+
+    width: parent.width
+    height: childrenRect.height
+
+    ExpandableCard
+    {
+        headerItem: Row
+        {
+            height: 48 * screenScaleFactor // TODO: Theme!
+            anchors.left: parent.left
+            anchors.leftMargin: 24 * screenScaleFactor // TODO: Theme!
+            spacing: 18 * screenScaleFactor // TODO: Theme!
+
+            MonitorPrintJobPreview
+            {
+                printJob: base.printJob
+                size: 32 * screenScaleFactor // TODO: Theme!
+                anchors.verticalCenter: parent.verticalCenter
+            }
+
+            Label
+            {
+                text: printJob && printJob.name ? printJob.name : ""
+                color: "#374355"
+                elide: Text.ElideRight
+                font: UM.Theme.getFont("medium") // 14pt, regular
+                anchors.verticalCenter: parent.verticalCenter
+                width: 216 * screenScaleFactor // TODO: Theme! (Should match column size)
+
+                // FIXED-LINE-HEIGHT:
+                height: 18 * screenScaleFactor // TODO: Theme!
+                verticalAlignment: Text.AlignVCenter
+            }
+            
+            Label
+            {
+                text: printJob ? OutputDevice.formatDuration(printJob.timeTotal) : ""
+                color: "#374355"
+                elide: Text.ElideRight
+                font: UM.Theme.getFont("medium") // 14pt, regular
+                anchors.verticalCenter: parent.verticalCenter
+                width: 216 * screenScaleFactor // TODO: Theme! (Should match column size)
+
+                // FIXED-LINE-HEIGHT:
+                height: 18 * screenScaleFactor // TODO: Theme!
+                verticalAlignment: Text.AlignVCenter
+            }
+
+            Item
+            {
+                anchors.verticalCenter: parent.verticalCenter
+                height: childrenRect.height
+                width: childrenRect.width
+
+                Label
+                {
+                    id: printerAssignmentLabel
+                    anchors.verticalCenter: parent.verticalCenter
+                    color: "#374355"
+                    elide: Text.ElideRight
+                    font: UM.Theme.getFont("medium") // 14pt, regular
+                    text: {
+                        if (printJob !== null) {
+                            if (printJob.assignedPrinter == null)
+                            {
+                                if (printJob.state == "error")
+                                {
+                                    return catalog.i18nc("@label", "Unavailable printer")
+                                }
+                                return catalog.i18nc("@label", "First available")
+                            }
+                            else
```

Comment:
This else isn't really needed, because it already returns in all other cases

Reply:
Fixed in 1de21c1